calendar_reboot.js

```diff
@@ -138,19 +135,29 @@ $(document).ready(function() {
         return true;
     };
     
-    var validateForm = function() {
-        console.log("validating the form");
+    var validateForm = function(startDateString, endDateString) {
+        
         
         var validateTitle = validateInput(calendarTitleId);
         var validateStartDate = validateInput(startDateId);
         var validateEndDate = validateInput(endDateId);
         
-        if (validateTitle && validateStartDate && validateEndDate) {
-            return true;
+        var isValid = validateTitle && validateStartDate && validateEndDate;
+        
+        if (isValid) {
```

Comment:
If you're returning a bool from inside an if statement, check to see if you can just return the condition variable itself.

```
if (x) {
  return true
} else {
  return false
}
```
is the same as `return x`.

Here you could do
```
if (isValid) {
  var validDates = validateDates(...)
  return validDates
} else {
  return false
}
```

or even just `return (isValid && validateDates(startDateString, endDateString))`